core/embed/lib/terminal.c

```diff
@@ -169,29 +171,16 @@ void term_print(const char *text, int textlen) {
 #endif
 }
 
-#ifdef TREZOR_EMULATOR
-#define mini_vsnprintf vsnprintf
-#include <stdio.h>
-#else
-#include "mini_printf.h"
-#endif
-
 // variadic term_print
 void term_printf(const char *fmt, ...) {
   if (!strchr(fmt, '%')) {
     term_print(fmt, strlen(fmt));
-#ifdef TREZOR_EMULATOR
-    printf("%s", fmt);
-#endif
```

Comment:
i think we want to keep these? so that the output goes to the virtual terminal and also to stdout

Reply:
This `printf` is not necessary here, as it's already in `unix/system.c`, which is a bit better location for this platform-specific code (and works event if panic/rsod handler is not registered or crashes).